src/services/importService.ts

```diff
@@ -200,21 +197,17 @@ export default class ImportService implements IImportService {
         const originalProject = JSON.parse(v1Project.content as string);
 
         for (const frameName in frameList) {
-            if (frameList.hasOwnProperty(frameName)) {
-                const frameRegions = frameList[frameName];
-                const asset = AssetService.createAssetFromFilePath(
-                `${v1Project.file.path.replace(/[^\/]*$/, "")}${frameName}`);
-                const assetState = originalProject.visitedFrames.indexOf(frameName) > -1 && frameRegions.length > 0
-                ? AssetState.Tagged : (originalProject.visitedFrames.indexOf(frameName) > -1
-                ? AssetState.Visited : AssetState.NotVisited);
-                const populatedMetadata = await assetService.getAssetMetadata(asset).then((metadata) => {
-                    const taggedMetadata = this.addRegions(metadata, frameRegions);
-                    taggedMetadata.asset.state = assetState;
-                    taggedMetadata.asset.path = `file:${taggedMetadata.asset.path}`;
-                    return taggedMetadata;
-                });
-                generatedAssetMetadata.push(populatedMetadata);
+            if (!frameList.hasOwnProperty(frameName)) {
+                continue;
             }
+            const frameRegions = frameList[frameName];
+            const asset = AssetService
+                .createAssetFromFilePath(`${v1Project.file.path.replace(/[^\/]*$/, "")}${frameName}`);
+            const assetState = this.getAssetState(originalProject, frameRegions, frameName);
+            generatedAssetMetadata.push(
+                await this.getPopulatedAssetMetadata(
+                    assetService, asset, assetState, frameRegions, undefined, true),
```

Comment:
`assetService` shouldn't need to be passed around.  We should be able to make it a instance member on the service.